addons/rose/.stylelintrc.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  extends: ['stylelint-config-standard-scss', 'stylelint-config-prettier-scss'],
```

Comment:
Used `stylelint-config-standard-scss` over `stylelint-config-sass-guidelines` which we're using in desktop. This one seems to be better supported now and is recommended by the official docs.